controllers/feeds/getAnonymousFeedChat.js

```diff
@@ -14,15 +14,24 @@ const {
 
 const getAnonymousFeedChatService = async (req, res) => {
   try {
+    let {last_fetch_date = null} = req.query;
+    if (last_fetch_date) {
+      last_fetch_date = new Date(last_fetch_date);
+      last_fetch_date = last_fetch_date.toISOString();
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conversion of `last_fetch_date` to an ISO string is unnecessary because MongoDB can compare Date objects directly. This could lead to potential issues with date format inconsistencies. It's better to keep the date as a Date object.
```diff
-    let {last_fetch_date = null} = req.query;
-    if (last_fetch_date) {
-      last_fetch_date = new Date(last_fetch_date);
-      last_fetch_date = last_fetch_date.toISOString();
-    }
+    const {last_fetch_date = null} = req.query;
+    if (last_fetch_date) {
+      last_fetch_date = new Date(last_fetch_date);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->